src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -433,6 +438,48 @@ class ComposterOverlay {
         return first ?: error("First is empty!")
     }
 
+    private fun getMaterials(internalName: String, itemName: String, itemsNeeded: Double) {
+        val sackData = fetchSackItem(internalName.asInternalName())
+        val amountInSacks = sackData.amount
+        val sacksLoaded = sackData.outdatedStatus
+        val having = InventoryUtils.countItemsInLowerInventory { it.getInternalName() == internalName.asInternalName() }
+
+        if (itemsNeeded.toInt() == 0 || itemName.removeColor() == "Biofuel") return
+
+        if (config.composterOverlayGetType == 0) {
```

Comment:
should be `if (config.composterOverlayGetType == 0 && !LorenzUtils.noTradeMode) {`